tests/foreman/ui/test_activationkey.py

```diff
@@ -1191,3 +1191,40 @@ def test_positive_delete_manifest(session):
         # Verify subscription is not assigned to activation key anymore
         ak = session.activationkey.read(activation_key.name, widget_names='subscriptions')
         assert not ak['subscriptions']['resources']['assigned']
+
+
+@pytest.mark.skip_if_not_set('clients', 'fake_manifest')
+@pytest.mark.tier3
+@pytest.mark.skipif((not settings.robottelo.repos_hosting_url), reason='Missing repos_hosting_url')
+def test_positive_ak_with_custom_product_on_rhel6(session, rhel6_contenthost, target_sat):
```

Comment:
If you're inheriting `target_sat`, it would be best to use it for entity and UI interactions in this test.